GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs

```diff
@@ -91,9 +91,9 @@ private void InitializeComponent()
             this.Ok.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Left | System.Windows.Forms.AnchorStyles.Right)));
             this.Ok.AutoSize = true;
             this.Ok.DialogResult = System.Windows.Forms.DialogResult.OK;
-            this.Ok.Location = new System.Drawing.Point(549, 3);
+            this.Ok.Location = new System.Drawing.Point(517, 3);
             this.Ok.Name = "Ok";
-            this.Ok.Size = new System.Drawing.Size(124, 25);
+            this.Ok.Size = new System.Drawing.Size(75, 25);
```

Comment:
Why?

Reply:
To make it consistent with the rest of the application I made the buttons the same size